docs/src/main/sphinx/release/release-477.md

```diff
@@ -0,0 +1,295 @@
+* Fix incorrect results with spatial joins. ({issue}`26021`)
+* Partition large pages to avoid `OutOfMemoryError` while serializing data in
+  the spooling protocol. ({issue}`25999`)
+* Fix spill files leaking in aggregation queries. ({issue}`26141`)
+* Reduce memory required for queries containing aggregations with a `DISTINCT`
+  or `ORDER BY` clause. ({issue}`26276`)
+* Fix memory tracking for ordered grouped aggregations. ({issue}`26276`)
+* Fix access control check when access is granted through groups in
+  `SET SESSION AUTHORIZATION`. ({issue}`26344`)
+* Fix failure when `geometry_to_bing_tiles` doesn't cover the full geometry
+  area, ensuring complete tile coverage. ({issue}`26459`)
+
+## Security
+
+## Web UI
+
+* Enhance UI responsiveness for Trino clusters without external network
+  access. ({issue}`26031`)
+* Add query details page to the Preview Web UI. ({issue}`25554`)
+* Add query JSON page to the Preview Web UI. ({issue}`26319`)
+
+## JDBC driver
+
+* Send detailed client information such as user-agent in the source.
+  ({issue}`25889`)
+
+## Docker image
+
+## CLI
+
+* Send detailed client information such as user-agent in the source.
+ ({issue}`25889`)
+* Allow configuring `--max-buffered-rows` and `--max-queued-rows`.
+  ({issue}`26015`)
+* Use actual stage id instead incremental number in progress output.
+  ({issue}`26139`)
+* Add support for keyboard navigation with {kbd}`Alt+↑` or {kbd}`Alt+↓` in query
+  history. ({issue}`26138`)
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Add support for using GCS without credentials. ({issue}`25810`)
+* Add ability to detect resource leakage in the runtime. ({issue}`26087`)
+* Add `azure.multipart-write-enabled` that enables multipart uploads for large
+  files. ({issue}`26225`)
+* Rename `s3.socket-read-timeout` config property to `s3.socket-timeout`. (
+  {issue}`26263`)
+* Prevent workers from going into full GC or crashing when decoding unusually
+  large Parquet footers. ({issue}`25973`)
+* Prevent creating multiple Alluxio client which can cause excessive resource
+  usage. ({issue}`26121`)
+* Release native filesystem resources/prevent leaks. ({issue}`26085`)
+* Fix failure when reading tables with `null` on `variant` type.
+  ({issue}`26016`)
+* Fix incorrect results when reading from Parquet files produced by old versions
+  of PyArrow. ({issue}`26058`)
+* Fix delta lake connector not closing resource streams properly. 
+  ({issue}`26092`)
+* Fix writing malformed checkpoint files when deletion vector is enabled.
+  ({issue}`26145`)
+* Fix failure when reading `null` values on `json` type columns.
+  ({issue}`26184`)
+* Fix skipping of row groups when the trino type is different from logical types
+  in case of Parquet files. ({issue}`26203`)
+* Reduce query failures from S3 throttling. ({issue}`26407`)
+* Fix incorrect results when the table uses deletion vectors and special
```

Comment:
How about : Fix incorrect results when table uses deletion vectors with special characters in partition path